pkg/runner/local_exec.go

```diff
@@ -283,3 +180,46 @@ func (*LocalExecutableRunner) ConfigType() reflect.Type {
 func (*LocalExecutableRunner) CompatibleBuilders() []string {
 	return []string{"exec:go"}
 }
+
```

Comment:
I thought there was a ticket to implement TerminateAll on local:exec somewhere, and wasn't done because the errgroup generally killed subprocesses anyway. Now, there are docker containers to take care of, so lets do it.

Reply:
Indeed. Since we use exec.CommandContext, the plan processes should be killed. But terminate commands are used to recover from broken states. To play it safe, we may want to find all processes started by the daemon process, and send them SIGKILL.